README.md

```diff
@@ -22,7 +22,10 @@ Want to contribute? Great! First, read [our guidelines for contributors](https:/
 
 We release by pushing the latest code to the master branch, followed by updating our [docs](https://developers.google.com/blockly) and [demo pages](https://blockly-demo.appspot.com). We typically release a new version of Blockly once a quarter (every 3 months). If there are breaking bugs, such as a crash when performing a standard action or a rendering issue that makes Blockly unusable, we will cherry-pick fixes to master between releases to fix them. The [releases page](https://github.com/google/blockly/releases) has a list of all releases.
 
-Releases are tagged by the release date (YYYYMMDD) with a leading '1.' and a trailing '.0' in case we ever need a major or minor version (such as [1.20190419.0](https://github.com/google/blockly/tree/1.20190419.0)). If you're using npm, a specific release can be installed by using its tag: `npm install git://github.com/google/blockly.git#1.20181219.0`
+Releases are tagged by the release date (YYYYMMDD) with a leading '1.' and a trailing '.0' in case we ever need a major or minor version (such as [1.20190419.0](https://github.com/google/blockly/tree/1.20190419.0)). If you're using npm, you can install the ``blockly`` package on npm: 
```

Comment:
Nit: They now have a leading 2. and will soon have a leading 3.

Reply:
I'll update to 3 then in anticipation for this next release.

Reply:
Actually since there's an example version, I'll just update to the latest version 2.20190722.1, and keep in mind to update this for the next major bump.